lib/dust.js

```diff
@@ -725,6 +725,11 @@
     return this;
   };
 
```

Comment:
I think you have the right separation. I think `mapThenable` is not something that a general public API would necessarily need, but `renderError` is nice. We can add it to the docs.